### ISSUE TITLE
Seeds doesn't reflect world generation.

### ISSUE THREAD

**Issue by daniktheboss** [iTXTech/Genisys#1759]
#### Problem
Ok, so when you join the world it generates a world... but that world is the only one! What I mean is that even if you use many worlds to create a new world using DIFFERENT seed.. it's the same! And also there are no ores! I have nothing installed on it.. did a fresh install of genisys and it's the same. No ores, and only one kind of world generation.

#### Steps to reproduce the issue
Idk... just join in the world and u will see a small pond nearby and some flat plain and mountains.

#### OS and versions
* Genisys: https://github.com/iTXTech/Genisys/commit/a1a6cecce85043fdfa560e672847b2f0c2a2baaa
* PHP: 7.0.6
* Server OS: linux
* Game version: 0.15.6

### Update
Used latest commit and no luck.

#### Crashdump, backtrace, memory dumps, 
None

**Comment by ishitatsuyuki** [comment]
Uh - yes, the PM random is sucked up. I will see if I can change that.

**Comment by daniktheboss** [comment]
And there are no ores... the most important thing!

**Comment by ZedCee** [comment]
Could we perhaps see your "worlds:" section from the bottom of "pocketmine.yml"?
I would like to try out the seeds you are using.  

Also the version you quoted is 2 months old, of which there have been many updates.  Searching back I am also unable to find the commit ala6cec in our history, PR #1170 not containing it and was also closed.  

For furture reference, Genisys is bleeding edge(updated to the minute) and thus there are major fixes every week, and the commit number is the version, not the codename/api.

Finally I would like to note: Blank seeds do not provide a random seed, they default to "deadbeef", so if upon generation without a preset seed, worlds will turn out the same.

**Comment by daniktheboss** [comment]
In the issue I updated saying I am using the latest update (yesterday last commit) and yes, I will send you a world so you can see there is zero ores in the world. Thanks

**Comment by ZedCee** [comment]
And I have read that.  "Latest commit" yesterday is not the "latest commit" today.  Please use commit number versions for clarity.  This prevents confusion for any issue that may be long standing.  

As for the line "From commit: ala6cec - Merge #1170 from Mattia990 to Master branch", that PR was never merged and not a recommended version to be running from. If you are not at all running from that version, then mentioning it is just confusing. If you are, I highly recommend against it.  There were lots of improvements to server stability and world generation in that 2 month time frame. (As well as some turbulence to World Gen)

I'm also not actually asking for a world itself, but for your world settings from `pocketmine.yml` located at the bottom under `worlds:`
I don't disbelieve that you have no ore in your worlds, I would however like to reproduce it on my own platform to see why that may be.

**Comment by dktapps** [comment]
https://github.com/iTXTech/Genisys/commit/a1a6cecce85043fdfa560e672847b2f0c2a2baaa

**Comment by ZedCee** [comment]
Thank you @dktapps for that.

Thank you @daniktheboss for updating your issue ticket.  The links make things concise and easy to navigate and read.  URLs are automatically truncated as you can see.
Also consider "latest" to be a taboo word and double check your quoted PRs (#), it can provoke confusion when looking back through commits.  Sorry for any confusion on that note from myself.

**Comment by daniktheboss** [comment]
worlds:
 #These settings will override the generator set in server.properties and allows loading multiple levels
 #Example:
 #world:
 # seed: 404
 # generator: FLAT:2;7,59x1,3x3,2;1;decoration(treecount=80 grasscount=45)
 this?

**Comment by ZedCee** [comment]
Those would be the settings.  Without a seed provided your world will default to "deadbeef"

Try adding:
```
world:
 seed: 1234567890
 generator: normal2
```
Rename world to whatever you want and randomize the seed however you feel.

**Comment by daniktheboss** [comment]
Ok I got different generations... but still when I use xray no ores... when I place a ore and try the xray it shows that ore

**Comment by dktapps** [comment]
We do not endorse the use of third-party mods or modified clients. If this is the only method by which you are testing this then this issue is invalid and you are wasting our time.

**Comment by daniktheboss** [comment]
I also dug in the ground with my friends (5) and we found nothing

**Comment by daniktheboss** [comment]
This isn't a mod thing. I used it to check if I have ores. I used it after mining with my friends and finding nothing.

**Comment by ZedCee** [comment]
I just generated a brand new world with no issues (in fact with better results than I had in the past).  I used TNT for a large scale excavation.

**Comment by daniktheboss** [comment]
Could it be a plugin fault?

**Comment by ZedCee** [comment]
Possibly. 
I recommend to move your plugin folder and if that fixes the issue, then incrementally install them back a few at a time to narrow down the problem plugin.
**Make this a staple part of your own debugging before posting an issue.**  If an issue cannot be reproduced with a brand new vanilla server set up, then something is wrong with your configuration.

**Comment by daniktheboss** [comment]
I tried doing a fresh install and still the same

**Comment by ZedCee** [comment]
I cannot reproduce this issue.  I am finding ores with no difficulty.  I use no mods and have a very small plugin list.  Typically I used pre-generated `anvil` maps, but for this issue I generated a couple brand new ones for you and found no issues.

If you are using a fresh install, make sure to delete (or move) the worlds folder and that all worlds are newly generated with the fresh install (goes without saying, already generated maps will not be populated with ores).

If this still causes problems, I recommend you come over to one of our chat networks(gitter.im, Mattermost, or IRC), to continue troubleshooting in real-time.
